src/App.module.css

```diff
@@ -73,6 +86,40 @@
     border-radius: 5px;
 }
 
+.petName {
+    margin-bottom: 5px;
+}
+
+.enterName {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
+}
+
+.inputEnterName {
+    width: auto;
+    margin-right: 5px;
+    padding-right: 5px;
+}
+
+.button {
+    margin-top: 5px;
+    margin-right: -50%;
+    width: 50px;
+}
+
+.text {
+    margin-top: 10px;
+    padding: 0;
+}
+
+/*.attention {*/
+/*    color: red;*/
+/*    font-size: 16px;*/
+/*}*/
+
```

Comment:
try to not leave comments in the code unless they are absolutely necessary, for example when they explain some concept in the code